src/js/player.js

```diff
@@ -4069,15 +4069,15 @@ Player.prototype.options_ = {
 
   // Included control sets
   children: [
+    'resizeManager',
```

Comment:
maybe revert this change if we need to re-trigger the event?

Reply:
OK, so I was going to remove this change and put resizeManager back at the bottom of the children list, but it breaks the controls for Safari. When I removed the `visibility: hidden;` css for the resize manager it made it so that the iframe was also intercepting other clicks on the video element. So we have to keep resizeManager at the top for controls to work correctly.

Reply:
If we still wanted to move it, I would put it after mediaLoader.